modules/chamber_params/main.tf

```
@@ -24,3 +24,22 @@ resource "aws_ssm_parameter" "ignored_values" {
     ]
   }
 }
+
+data "aws_ssm_parameter" "this" {
+  for_each = aws_ssm_parameter.this
+  name     = "/${lower(var.group)}/${lower(each.key)}"
+}
+
+locals {
+  ignored_value_map = [
+    for k in data.aws_ssm_parameter.this : k.value
+  ]
+  concat_string = join("", local.ignored_value_map)
+  result        = md5(local.concat_string)
+  sensitive     = true
+}
+
+output "hash" {
+  value     = local.result
+  sensitive = true
```

```suggestion
  value = md5(join("", [for k in data.aws_ssm_parameter.this : k.value]))
```

Shorter. Just move it to `output.tf`